doc/man5/srec_efinix.5

```diff
@@ -0,0 +1,37 @@
+'\" t
+.\"     srecord - manipulate eprom load files
+.\"     Copyright (C) 2003, 2005-2009 Peter Miller
+.\"
+.\"     This program is free software; you can redistribute it and/or modify
+.\"     it under the terms of the GNU General Public License as published by
+.\"     the Free Software Foundation; either version 3 of the License, or
+.\"     (at your option) any later version.
+.\"
+.\"     This program is distributed in the hope that it will be useful,
+.\"     but WITHOUT ANY WARRANTY; without even the implied warranty of
+.\"     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.\"     GNU General Public License for more details.
+.\"
+.\"     You should have received a copy of the GNU General Public License
+.\"     along with this program. If not, see
+.\"     <http://www.gnu.org/licenses/>.
+.\"
+.ds n) srec_efinix
+.TH \*(n) 5 SRecord "Reference Manual"
+.SH NAME
+srec_efinix \- Efinix bitstream file format
+.if require_index \{
+.XX "srec_efinix(5)" "Efinix bitstream file format"
+.\}
+.SH DESCRIPTION
+This format is the Efinix bitstream file format.
+This is a hex format generated by the IDE from Efinix to program their FPGAs.
+
+The text file to be read contains two hex-chars on each line.
```

Comment:
Just for clarity, can I confirm that what we are describing is two bytes on each line? Based on the example you included in your test this would like something like this (where 0x0A is the newline/linefeed):

0x46, 0x42, 0x0A,
0x30, 0x42, 0x0A,
0x44, 0x30, 0x0A,
.
.
.

If that is the case, for clarity, can we please refer to this as
`contains two bytes followed by a 0x0A representing an ASCII newline.`

Reply:
It's one byte (in binary output) for each line in efinix input. 
Each line has two hex symbols (two nibbles = one byte) in ASCII and a newline.

Reply:
Understood. Thanks for clarifying